ddapm_test_agent/agent.py

```diff
@@ -801,7 +806,8 @@ def make_app(
             CheckTraceCountHeader,
             CheckTraceContentLength,
             CheckTraceStallAsync,
-        ],
+        ]
+        + additional_check_classes,
```

Comment:
having 3 different ways to specify or not specify checks is getting a bit complex IMO. I wonder if we should just have the user of the test agent specify the checks they want and avoid having any defaults.

WDYT @wconti27 @devinsba?

Reply:
IMO I'm okay with this approach as it solves all requirements and I don't see needing any further complexity in the future. Having the user specify every check name to run in an env variable will lead to a very long variable value once we have 10+ checks.

Reply:
I think the verbosity is a feature as it becomes obvious what checks are enabled and it's obvious how to add/remove them.

@wconti27 maybe a file configuration is better suited to help the readability issue?

Reply:
Good point, having to explicitly specify checks does seem a lot more straightforward. Given that we don't have too many checks right now, Im fine with adding the config file as a possible future issue/improvement and just using an ENV variable for now. Would be great if we can get this merged soon in order to unblock the new service naming checks for Java/Python!

Reply:
I'll rework it to make it completely opt in. I will create a PR to revert the previous PR and open a new one that reworks the whole thing